charts/ingress-nginx/templates/controller-ingressclass.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if and (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion) (.Values.controller.ingressClassResource.enabled) -}}
+apiVersion: networking.k8s.io/v1
+kind: IngressClass
+metadata:
+  labels:
+    {{- include "ingress-nginx.labels" . | nindent 4 }}
+    app.kubernetes.io/component: controller
+    {{- with .Values.controller.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  name: {{ .Values.controller.ingressClass }}
+{{- if .Values.controller.ingressClassResource.default -}}
```

Comment:
```suggestion
{{- if .Values.controller.ingressClassResource.default }}
```

Reply:
Minor bug. When you set default to true, the identation brings the `annotations` block next to the `{{ .Values.controller.ingressClass }}` definition.

So, let's say my ingress class' name is `nginx`, the renderer output would show:

```yaml
...................
    app.kubernetes.io/component: controller
  name: nginxannotations:      <------------------------------------ BUG
    ingressclass.kubernetes.io/is-default-class: "true"
spec:
  controller: k8s.io/ingress-nginx
...........................
```

Reply:
Thanks for your feedback. I will have a look on this

Reply:
Done, I applied it directly instead of using the Github GUI for a cleaner commit log.